rollup-interface/src/state_machine/mod.rs

```diff
@@ -23,10 +23,10 @@ pub trait BasicAddress:
     + Send
     + Sync
     + Clone
-    + std::hash::Hash
+    + core::hash::Hash
```

Comment:
This is exactly the same pattern I'm suggesting we also use for `Vec`, `Box`, `String`, etc.. These types are equivalent and more portable.

Reply:
Let's centralize the discussion on the other comment